src/index.ts

```diff
@@ -192,8 +192,7 @@ function teenyRequest(
   const multipart = reqOpts.multipart as RequestPart[];
   if (reqOpts.multipart && multipart.length === 2) {
     if (!callback) {
-      console.error('multipart without callback not implemented.');
-      return;
+      throw new Error('multipart without callback not implemented.');
```

Comment:
The wording here tripped me up, but I believe this is a "TODO"/missing feature of the library. If that sounds right, could you either add in a "@TODO" here, or a new issue to request support for multipart uploads through streaming/promises?